finch/src/services/logger.js

```diff
@@ -0,0 +1,32 @@
+import config from '../../config/index.js';
+
+export function logInfo(msg, context = {}) {
+  /* eslint-disable-next-line no-console */
+  console.log(formatLog({
+    type: 'info',
+    msg,
+    context,
+  }));
+}
+
+export function logError(msg, context = {}) {
+  /* eslint-disable-next-line no-console */
+  console.error(formatLog({
+    type: 'error',
+    msg,
+    context,
+  }));
+}
+
+export function logWarning(msg, context = {}) {
+  /* eslint-disable-next-line no-console */
+  console.warn(formatLog({
+    type: 'warning',
+    msg,
+    context,
+  }));
+}
+
+export function formatLog(logContents) {
+  return config.logs.prettify ? logContents : JSON.stringify(logContents);
```

Comment:
You may want to use `JSON.stringify(logContents, null, 4)` for prettified logs. Otherwise `console.log` will print nested objects as `Object [Object]`

Reply:
cool, will do this